docs/IntegrationWithExistingApps.md

```diff
@@ -439,7 +440,7 @@ import React
 
   let rootView = RCTRootView(
       bundleURL: jsCodeLocation,
-      moduleName: "RNHighScores",
+      moduleName: "MyReactNativeApp",
```

Comment:
Ok, I agree that there's a bug to fix it, but it's already fixed in another merge request. Compare with: https://github.com/facebook/react-native/pull/15252/files